### PR TITLE
Clarify requirement of having etcd group

### PR DESCRIPTION
**Pull request by sdodson**

### CODE REVIEW

**Comment by sdodson** [comment]
I'll update the check so that it accepts either 1 or 3 group members.

File: playbooks/common/openshift-cluster/evaluate_groups.yml [hunk 1]
```diff
@@ -43,11 +43,14 @@
   - name: Evaluate groups - Fail if no etcd hosts group is defined
     fail:
       msg: >
-        No etcd hosts defined. Running an all-in-one master is deprecated and
-        will no longer be supported in a future upgrade.
+        No etcd hosts defined. Running embedded etcd is no longer be supported.
```
Comment (abutcher) [inline playbooks/common/openshift-cluster/evaluate_groups.yml]: Running embedded etcd is no longer supported. **

Comment (sdodson) [reply playbooks/common/openshift-cluster/evaluate_groups.yml]: better?

Comment (abutcher) [reply playbooks/common/openshift-cluster/evaluate_groups.yml]: Nice

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[test]

**Pull request merged**